bsb/morphologies/__init__.py

```diff
@@ -1475,6 +1488,72 @@ def ceil_arc_point(self, arc):
                 return i
         return len(self) - 1
 
+    def get_axial_distance(self, idx_start=0, idx_end=-1, return_max=False):
```

Comment:
```suggestion
    def get_axial_distances(self, idx_start=0, idx_end=-1, return_max=False):
```

Since it returns multiple distances :] Sorry! 😇